src/style.css

```
@@ -1,3 +1,66 @@
+:root {
+  --secondarycolor: #00acdc;
+  --primarycolor: #000;
+}
+
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
+.container {
+  display: flex;
+  box-sizing: border-box;
+  width: 100%;
+  justify-content: space-between;
+  align-items: center;
+  padding: 10px 50px;
```

We need to simplify all your margins and paddings. I think we should stars with padding here. We can get rid of padding top & bottom here, so it doesn't determine the height of the entire header
```suggestion
  padding: 0 50px;
```